Nikita Korzhavin/flex/index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.css"
+    />
+    <link rel="stylesheet" href="./cubes.css" />
+    <title>Cubes[1]</title>
+  </head>
+  <body>
+    <section class="bg1">
+      <div class="bg2">
+        <article class="widget">
+          
+            <div class="widget__card card card_bg-color_06b3db">
+              <div class="card__img card__img_size_l card__img_opacity">
+                <img src="./images/cloud-rain.svg" alt="cloud-rain" />
+              </div>
+              <p class="card__text card__text_position_1 card__text_size_m">CHICAGO / 23&deg;</p>
+            </div>
+
+            <div class="widget__card card card_bg-color_4ed486">
+              <div class="card__img card__img_size_l card__img_opacity">
+                <img src="./images/fog.svg" alt="Fog" />
+               </div>
+              <p class="card__text card__text_position_1 card__text_size_m">SEATTLE / 34&deg;</p>
+            </div>
+
+            <div class="widget__card card card_bg-color_6139f6">
+              <div class="card__img card__img_size_l card__img_opacity">
+                <img src="./images/cloud-sun.svg" alt="CS" />
```

Comment:
`CS` is not very meaningful `alt` for the image. Remember that user should be able to understand it if the picture won't load.

Reply:
fixed (in commit)